tests/integration/targets/clickhouse_role/tasks/initial.yml

```
@@ -33,12 +33,15 @@
   community.clickhouse.clickhouse_role:
     state: present
     name: test_role
+    settings:
+      - max_memory_usage = 15000 READONLY
+      - max_memory_usage_for_all_queries = 15000 MIN 15000 MAX 16000 WRITABLE
 
 - name: Test 2 - Check the return values
   ansible.builtin.assert:
     that:
     - result is changed
-    - result.executed_statements == ['CREATE ROLE test_role']
+    - result.executed_statements != []
```

```suggestion
    - result.executed_statements != []
```
is it an issue to put the executed query here?

---

The query will be long, I decided not to specify it.

---

+1 to including the query no matter the length